towerlib/towerlib.py

```diff
@@ -1399,21 +1399,15 @@ def create_credential_in_organization(self,  # pylint: disable=too-many-argument
 
         Raises:
             InvalidOrganization: The organization provided as argument does not exist.
-            InvalidUser: The user provided as argument does not exist.
-            InvalidTeam: The team provided as argument does not exist.
             InvalidCredentialType: The credential type provided as argument does not exist.
             InvalidVariables: The inputs provided as argument is not valid json.
 
         """
         organization_ = self.get_organization_by_name(organization)
         if not organization_:
             raise InvalidOrganization(organization)
-        user_ = self.get_user_by_username(user)
-        if not user_:
-            raise InvalidUser(user)
-        team_ = organization_.get_team_by_name(team)
-        if not team_:
-            raise InvalidTeam(team)
+        user_id = self.get_user_by_username(user).id if user else None
```

Comment:
I think that we should still raise an exception if a user provides a username or a team and they are invalid. If they don't we don't have to check and that is fine but if they provide one and it is wrong we should not magically fix that for them.

Reply:
Added the logic back to raise the exeception